### DISCUSSION TITLE
How is "Discharge capacity [A.h]" calculated?

### DISCUSSION THREAD

**Discussion by emrebaburoglu** [pybamm-team/PyBaMM#2198] [Q&A] [answered]
Hello, 

I'm trying to simulate battery ageing/rate capability tests in lithium ion half-cells and am having some issues understanding the results. When I do a rate capability test, I see that the "Discharge capacity [A.h]" decreases with increasing C-rate but the 'Positive electrode capacity [A.h]' stays the same even though it's a half-cell (negative is lithium). Likewise, when I simulate active material loss, I get that the "X-averaged positive electrode active material volume fraction" decreases over cycles but the discharge capacity stays the same. Is there any documentation for me to read to see how "Discharge capacity [A.h]" is calculated and what parameters it depends on?

**Comment by valentinsulzer** [comment, marked as answer]
Discharge capacity [A.h] is just the time-integral of the current in A.

**Reply by emrebaburoglu** [reply]
and how is the current calculated/what parameters does it depend on?
